documentation/openapi.json

```diff
@@ -1409,44 +1241,129 @@
         "additionalProperties": false,
         "description": "Represents a partial model when enumerating all operations."
       },
-      "OperationError": {
+      "ProblemDetails": {
         "type": "object",
```

Comment:
Are these breaking changes in the api spec or did everything just get shuffled around a lot?